evm/pom.xml

```diff
@@ -1,20 +1,48 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.horizen</groupId>
   <artifactId>evm</artifactId>
   <version>0.1.0-SNAPSHOT</version>
-  <inceptionYear>2022</inceptionYear>
   <name>${project.groupId}:${project.artifactId}</name>
   <description>This library provides access to a standalone version of the go-ethereum EVM and its state storage layer StateDB and underlying LevelDB.</description>
+  <url>https://github.com/${project.github.organization}/${project.artifactId}</url>
+  <inceptionYear>2022</inceptionYear>
+  <licenses>
+    <license>
+      <name>MIT License</name>
+      <url>https://opensource.org/licenses/MIT</url>
+    </license>
+  </licenses>
+  <developers>
+    <developer>
+      <name>Zen Blockchain Foundation</name>
+      <email>info@zensystem.io</email>
```

Comment:
@cronicc @ysibirski is it expected to publish this lib under ZBF? Or do we expect it to be published as a part of HL?

Reply:
Any repository stored on HorizenOfficial should publish to ZBF.